### DISCUSSION TITLE
upload plugin - set url of image

### DISCUSSION THREAD

**Discussion by maximiliano-gt** [strapi/strapi#6959] [Legacy Topics]
Hello, when I do an API call of my articles, and having an image on it the response is like:

`{...........
      "image": {
                       .......
                      "url": "/uploads/photo.jpeg",
                      ......}}`

How do I config strapi in order to have a complete url like  "url": "MYWEBSITE.COM/uploads/photo.jpeg".

This also happens when using the Rich Text field, and trying to add an image inside the slate editor:

`![phot](/uploads/photo.jpeg)` and i need `![photo](MYWEBSITE.COM/uploads/photo.jpeg)`

**Comment by macorifice** [comment]
I have the same request for my project because I've installed my content and Strapi app in two different host

**Reply by sandiprb** [reply]
@maximiliano-gt  @macorifice  I think you can take care of this on your front-end application it self.

Prefix your image src URLs with your API endpoints 

React example  as below
```
<img src={`${STRAPI_URL}/uploads/photo.jpeg`}  />
```
